docs/utilities/parameters.md

```diff
@@ -238,183 +332,181 @@ The AWS Systems Manager Parameter Store provider supports two additional argumen
-        AppConfigDataClient appConfigDataClient = AppConfigDataClient.builder()
-                .httpClientBuilder(UrlConnectionHttpClient.builder())
-                .region(Region.EU_CENTRAL_2)
-                .build();
+    import software.amazon.lambda.powertools.parameters.appconfig.AppConfigProvider;
 
-        // Get an instance of the DynamoDbProvider
-        AppConfigProvider appConfigProvider = ParamManager.getAppConfigProvider(appConfigDataClient, "my-environment", "my-app");
+    public class RequestHandlerWithParams implements RequestHandler<String, String> {
     
-        // Retrieve a single parameter
-        String value = appConfigProvider.get("my-key"); 
-    } 
+        // Get an instance of the SecretsProvider. We can provide a custom client here if we want,
+        // for instance to use a particular region.
+        AppConfigProvider appConfigProvider = AppConfigProvider
+                .builder()
+                .withClient(AppConfigDataClient.builder().build())
+                .build();
+    
+        public String handleRequest(String input, Context context) {
+            // Retrieve a single secret
+            String value = appConfigProvider
+                    .get("/my/secret");
+    
+            // Return the result
+            return value;
+        }
+    }    
     ```
 
-
 ## Advanced configuration
 
 ### Caching
+Each provider uses the CacheManager to cache parameter values. When a value is retrieved using from the provider, a 
+custom cache duration can be provided using `withMaxAge(duration, unit)`. 
 
-By default, all parameters and their corresponding values are cached for 5 seconds.
+If this is not specified, the default value set on the CacheManager itself will be used. This default can be customized
+by calling `setDefaultExpirationTime(duration, unit)` on the CacheManager.
 
-You can customize this default value using `defaultMaxAge`. You can also customize this value for each parameter using 
-`withMaxAge`.
+=== "Customize Cache - Per Item"
 
-=== "Provider with default Max age"
-
-    ```java hl_lines="9"
-    import software.amazon.lambda.powertools.parameters.SecretsProvider;
-    import software.amazon.lambda.powertools.parameters.ParamManager;
-
-    public class AppWithSecrets implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
-        // Get an instance of the Secrets Provider
-        SecretsProvider secretsProvider = ParamManager.getSecretsProvider()
-                                                      .defaultMaxAge(10, ChronoUnit.SECONDS);
-
-        String value = secretsProvider.get("/my/secret");
+    ```java hl_lines="14 15"
+    import java.time.Duration;
+    import software.amazon.lambda.powertools.parameters.appconfig.AppConfigProvider;
+    import software.amazon.lambda.powertools.parameters.cache.CacheManager;
+    
+    public class CustomizeCacheTimeout {
+        
+        public void CustomizeCache() {
```

Comment:
do we need that ?

Reply:
Dropped as the single complete example covers this